xarray/util/generate_aggregations.py

```diff
@@ -218,7 +287,15 @@ def {method}(
     example="""\n
         Specify ``ddof=1`` for an unbiased estimate.
 
-        >>> {calculation}(skipna=True, ddof=1)""",
+        >>> {calculation}(skipna=True, ddof=1)
+        """,
+)
+
+axis = ExtraKwarg(
+    docs=_AXIS_DOCSTRING,
+    kwarg="axis: int | Sequence[int] | None = None,",
+    call="axis=axis,",
+    example="",
```

Comment:
I initially thought this was just a documentation error with the old cumsum/prod docs where they had axis as a parameter but the current version of dataarray.cumsum/dataarray.cumprod does support axis input. I was worried that removing it may break some users' code. Is this something we should depreciate, leave in, or just outright remove?

I also just realized that I had axis as an additional kwarg for the dataset methods not dataarray methods which is my mistake.

Reply:
>  old cumsum/prod docs where they had axis

We've been removing these so IMO its OK. I think it was an issue with the older reduction functions too